benchmarks/backend_request_func.py

```diff
@@ -224,10 +224,9 @@ async def async_request_openai_completions(
     pbar: Optional[tqdm] = None,
 ) -> RequestFuncOutput:
     api_url = request_func_input.api_url
-    assert api_url.endswith(
-        "completions"
-    ), "OpenAI Completions API URL must end with 'completions'."
-
+    # assert api_url.endswith(
+    #     "completions"
+    # ), "OpenAI Completions API URL must end with 'completions'."
```

Comment:
I actually feel we could just remove this assert...

Reply:
removed the changes for profiler. Now that https://github.com/vllm-project/vllm/pull/6883 is merged I will probably make a PR add torch profiler support to everything. As now it becomes really tedious to add manually